Fabric.Authorization.Persistence.SqlServer/Stores/EDW/SecurityContext.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Fabric.Authorization.Domain.Models.EDW;
+using Fabric.Authorization.Persistence.SqlServer.Configuration;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fabric.Authorization.Persistence.SqlServer.Stores.EDW
+{
+    public class SecurityContext : DbContext, ISecurityContext
+    {
+        protected readonly ConnectionStrings connectionStrings;
+
+        public DbSet<EDWIdentity> EDWIdentities { get; set; }
+        public DbSet<EDWRole> EDWRoles { get; set; }
+        public DbSet<EDWIdentityRole> EDWIdentityRoles { get; set; }
+
+        public SecurityContext() { }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(connectionStrings.EDWAdminDatabase);
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+            //if (modelBuilder == null)
+            //{
+            //    return;
+            //}
+
+            //modelBuilder.Entity<EDWIdentityRole>()
+            //    .HasKey(identityRole => new { identityRole.IdentityId, identityRole.RoleId });
+            //modelBuilder.Entity<EDWIdentityRole>()
+            //    .HasOne(identityRole => identityRole.EDWRole)
+            //    .WithMany(role => role.EDWIdentityRoles)
+            //    .HasForeignKey(identityRole => identityRole.RoleId);
+            //modelBuilder.Entity<EDWIdentityRole>()
+            //    .HasOne(identityRole => identityRole.EDWIdentity)
+            //    .WithMany(role => role.EDWIdentityRoles)
+            //    .HasForeignKey(identityRole => identityRole.RoleId);
+        }
+
+        public ISecurityContext CreateContext()
```

Comment:
If we provide the appropriate constructor that takes in a ConnectionStrings object we should be able to get rid of this method and simply allow the DI system to create the context for us, then we also don't have to worry about disposing it, the lifetime will get managed by the DI system.